packages/fxa-content-server/server/lib/routes/validate-email-domain.js

```diff
@@ -57,7 +57,7 @@ const tryResolveWith = (resolveFunc) => async (domain) => {
   }
 };
 
-module.exports = function () {
+module.exports = function (config) {
```

Comment:
We need to update the existing tests to pass a config object.  An example of an existing test failing:

```
× node - routes/validate-email-domain - route.process - responds with {result: "MX"} when there is a MX record
TypeError: Cannot read property 'get' of undefined
  at Object.process @ server/lib/routes/validate-email-domain.js:71:37
  at Test.responds with {result: "MX"} when there is a MX record [as test] @ tests/server/routes/validate-email-domain.js:55:22
  @ ../../src/lib/Test.ts:268:51
```

Reply:
@chenba Let me update the routes.js test file. After that everyting is going to be fine I guess :)

Reply:
@chenba here we have provided a config object to the complete function so each test is going to break since all the other test does not have the config object

Reply:
Yes.  Please update the tests that are now failing due to the changes introduce in this PR.  Thanks!

Reply:
@chenba  Commiting the new changes.